sys-block/tcmu-runner/tcmu-runner-1.4.1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils linux-info
+
+DESCRIPTION="A daemon that handles the userspace side of the LIO TCM-User backstore"
+HOMEPAGE="http://www.open-iscsi.com/"
```

Comment:
Please link to TLS based connections whenever possible, e.g. https://www.open-icsci.com.